### ISSUE TITLE
CRSF protocol support

### ISSUE THREAD

**Issue by controlflow** [betaflight/betaflight-tx-lua-scripts#44]
I really miss BF lua scripts when I use TBS Crossfire quads...
Now with CRSF protocol support built-in in OpenTX 2.x I wonder if it would be possible to abstract this script from telemetry protocol implementation...

**Comment by si618** [comment]
[Already done](https://github.com/codecae/betaflight-tx-lua-revised).  Hopefully fork will get merged back here.

[Note F3 issue](https://github.com/codecae/betaflight-tx-lua-revised/issues/9) is waiting on pull request to ßF master.

**Comment by codecae** [comment]
The code now supports Crossfire!  For best results use Betaflight v3.2.2 and Crossfire 2.11 or greater.
